### ISSUE TITLE
Suggestion: Master/details views

### ISSUE THREAD

**Issue by tilleul** [formtools/core#347]
EDIT SUBMISSIONS:
Would it be possible to add the ability to display detailed info (coming from a view) when selecting an item in a dropdown box ?

Let's say I have a form for my clients, including Name, address, phone, etc.
In another form (invoices for instance), I have a dropdown field linked to the clients form/table. It would be great if, as an option, the field is also linked to a view to the clients form. Then, whenever a client is selected in the dropdown, the engine would display (directly behind it and just before the next field) the view for the selected client.

SUBMISSIONS LIST:
Maybe a similar principle could be applied here: fields linked to other forms could display a "sub-view" instead of just the label ...

FILTERS and DATA VISUALISATION:
Then even greater would be the ability to filter on those linked forms... For example, if I want to see all invoices that have clients living in New York we could have additional options/dropdowns to select fields in the "details" form ....

**Comment by benkeen** [comment]
Yeah absolutely. Being able to link forms/data in a manner like this has been on the feature request list for a long time - I've just been focusing on bigger fish these last few years. Thanks for posting this, good to get a reminder.
